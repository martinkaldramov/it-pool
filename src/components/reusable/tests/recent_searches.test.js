import React from 'react';
import { shallow } from 'enzyme';
import RecentSearches from '../recent_searches';
import MockProfile from '../../../mock_data/mock_profile';

describe('RecentSearches', () => {
  const wrap = shallow(<RecentSearches recentSearches={MockProfile.recentSearches} />);

  it('should render successful', () => {
    expect(wrap).toMatchSnapshot();
  });
});
