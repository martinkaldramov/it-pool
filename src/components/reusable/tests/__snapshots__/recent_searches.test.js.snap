// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecentSearches should render successful 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <RecentSearches
    searches={
        Array [
            "Web Developer",
            "linux system administrator",
            "nodejs backend developer",
            "ReactJS Developer",
            "network engineer",
            "devops",
            "C# desktop developer",
          ]
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <li>
          Web Developer
</li>,
        <li>
          linux system administrator
</li>,
        <li>
          nodejs backend developer
</li>,
        <li>
          ReactJS Developer
</li>,
        <li>
          network engineer
</li>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Web Developer",
        },
        "ref": null,
        "rendered": "Web Developer",
        "type": "li",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "linux system administrator",
        },
        "ref": null,
        "rendered": "linux system administrator",
        "type": "li",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "nodejs backend developer",
        },
        "ref": null,
        "rendered": "nodejs backend developer",
        "type": "li",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "ReactJS Developer",
        },
        "ref": null,
        "rendered": "ReactJS Developer",
        "type": "li",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "network engineer",
        },
        "ref": null,
        "rendered": "network engineer",
        "type": "li",
      },
    ],
    "type": "ul",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <li>
            Web Developer
</li>,
          <li>
            linux system administrator
</li>,
          <li>
            nodejs backend developer
</li>,
          <li>
            ReactJS Developer
</li>,
          <li>
            network engineer
</li>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Web Developer",
          },
          "ref": null,
          "rendered": "Web Developer",
          "type": "li",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "linux system administrator",
          },
          "ref": null,
          "rendered": "linux system administrator",
          "type": "li",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "nodejs backend developer",
          },
          "ref": null,
          "rendered": "nodejs backend developer",
          "type": "li",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "ReactJS Developer",
          },
          "ref": null,
          "rendered": "ReactJS Developer",
          "type": "li",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "network engineer",
          },
          "ref": null,
          "rendered": "network engineer",
          "type": "li",
        },
      ],
      "type": "ul",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
